docs/source/special.rst

```diff
@@ -44,3 +40,20 @@ Functions
 .. autofunction:: xlog1py
 .. autofunction:: xlogy
 .. autofunction:: zeta
+
+Error Functions and and Exponential Integrals
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Error Functions
```

Comment:
Cool

Reply:
Yeah! I want to make this page sharp. 👨‍🎨